### DISCUSSION TITLE
Any issues with macOS Sequoia?

### DISCUSSION THREAD

**Discussion by clanger9** [ExistentialAudio/BlackHole#810] [Q&A] [answered]
Simple question: is anyone successfully using blackhole-2ch on macOS Sequoia?
I’d like to get confirmation that everything is OK before I upgrade my M1 Mac…

**Comment by devinroth** [comment, marked as answer]
Yeah it’s working fine.

Devin

**Reply by clanger9** [reply]
Fantastic, thank you for the confirmation!

**Reply by clanger9** [reply]
I can confirm: no issues on macOS Sequoia for me 👍

**Comment by ccgr33n** [comment]
Thank you - as well.
